src/components/Menu/Menu.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+
+//Axios
+import axios from 'axios';
+import { MenuURL } from '../../api/ApiLinks';
+
+//Icons
+import Star from '../../icons/star-svgrepo-com.svg';
+import Info from '../../icons/info-circle-svgrepo-com.svg';
+
+import classes from './menu.module.scss';
+
+const Menu = () => {
+  const [menuList, setMenuList] = useState([]);
+
+  useEffect(() => {
+    axios.get(MenuURL).then((response) => setMenuList(response.data));
+  }, []);
+
+  return (
+    <div>
+      {menuList.map((restaurants) => (
+        <div key={restaurants._id}>
+          {restaurants._id === '651e32d67a5bb9de20aaf7ec' ? (
```

Comment:
No harded  code the ID  and follow the main branch folder structure eg: Login Componement